server/belga/publish/belga_newsml_1_2.py

```diff
@@ -436,10 +445,31 @@ def _format_picture(self, newscomponent_1_level, picture):
             )
             contentitem = SubElement(newscomponent_3_level, 'ContentItem')
             SubElement(contentitem, 'Format', {'FormalName': 'Text'})
-            SubElement(contentitem, 'DataContent').text = picture.get(key)
+
+            if key == 'description_text' \
+                    and picture.get('extra', {}).get('people') \
+                    and picture.get('extra', {}).get('event_description'):
+                # format caption using `people` and `event_description` fields
+                data_content = '{} on the picture regarding {}'.format(
+                    picture['extra']['people'], picture['extra']['event_description']
+                )
+            elif key == 'description_text' \
+                    and picture.get('extra', {}).get('people') \
+                    and not picture.get('extra', {}).get('event_description'):
```

Comment:
I would say it's not needed here as if that part didn't pass the `if` branch would run